h2o-py/h2o/frame.py

```diff
@@ -2568,6 +2567,9 @@ def isax(self, num_words, max_cardinality, optimize_card=False, **kwargs):
 
     def convert_H2OFrame_2_DMatrix(self, predictors, yresp, h2oXGBoostModel):
         '''
+        This method will require to import the following toolboxes:
+        xgboost, pandas, numpy and scipy.sparse.
+
```

Comment:
This method requires that you import the following toolboxes:

Reply:
Done